### ISSUE TITLE
Renal artery aneurysm

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6467]
**Preferred term label:**
Renal artery aneurysm

**Synonyms**

**Definition (free text, please give PubMed ID)**
a dilated segment of renal artery that exceeds twice the diameter of a normal renal artery.
Increase in diameter of a segment of the renal artery. Various definitions exist in the literature. Focal, isolated dilatation of all three layers of the arterial wall over 1.5 times the diameter of the disease-free proximal adjacent arterial segment. PMID: 25537277

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0008776

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:615688 Vasculitis, autoinflammation, immunodeficiency, and hematologic defects syndrome
PMID: 27059682  3/15

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
